dotcom-rendering/src/web/layouts/ImmersiveLayout.tsx

```diff
@@ -380,7 +382,37 @@ export const ImmersiveLayout = ({
 									webTitle={CAPI.webTitle}
 									ajaxUrl={CAPI.config.ajaxUrl}
 								/>
-								{showBodyEndSlot && <div id="slot-body-end" />}
```

Comment:
We have [some styles](https://github.com/guardian/dotcom-rendering/blob/8f71e12d717f118d095834362f0fed58ea77ee3b/dotcom-rendering/src/web/components/SignInGate/gateDesigns/shared.tsx) that get applied against this id. It's possible that there's another div with the same id but if not then we might need to think of a way to keep those styles

Reply:
As `id` attributes are meant to be unique in valid HTML, I’ll wager that you’re right.

Let me investigate a bit more.